### DISCUSSION TITLE
Object Design of the Requirements of a Project

### DISCUSSION THREAD

**Discussion by emilcan-arican** [bounswe/bounswe2020group6#258] [General]
How should be the design of the requirements part of the projects? 

> Backend will return the requirements object that is sent to the database.
> Therefore, frontend and mobile teams should be coherent during implementations.

Mobile and Frontend teams should discuss and decide the format of the requirements part of a project.

@UkcaGreen @cemreefe @aliramazanmert @dogruomerfaruk 
@Tubagc @cihatkapusuz @DogukanKalkan @gulperii

**Comment by cemreefe** [comment]
I believe after our discussion with Alper Ahmetoğlu we decided to keep it as a multi-line text field.
